bedrock/firefox/templates/firefox/set-as-default/landing.html

```diff
@@ -16,24 +16,40 @@
   {{ css_bundle('firefox-default-landing') }}
 {% endblock %}
 
+{% if switch('set-as-default-experiment', 'en-US') %}
+  {% block js %}
```

Comment:
Sorry I didn't spot this the first time, but this should be the experiments block:
```suggestion
  {% block experiments %}
```

We also typically put the switch inside the block, though I'm not sure it makes a difference in this case.

Reply:
Oh good to know - I'd rather it be consistent anyways